packages/api/tests/endpoints/test-pdrs.js

```diff
@@ -222,14 +234,25 @@ test('default returns list of pdrs', async (t) => {
 });
 
 test('GET returns an existing pdr', async (t) => {
+  const pdrName = `${randomString()}.PDR`;
+  const newPGPdr = fakePdrRecordFactory({
+    name: pdrName,
+    status: 'completed',
+    collection_cumulus_id: t.context.collectionCumulusId,
+    provider_cumulus_id: t.context.providerCumulusId,
+    execution_cumulus_id: t.context.executionCumulusId,
+  });
+
+  // create a new PDR in RDS
+  await t.context.pdrPgModel.create(t.context.knex, newPGPdr);
+
   const response = await request(app)
-    .get(`/pdrs/${fakePdrs[0].pdrName}`)
+    .get(`/pdrs/${pdrName}`)
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
-  const { pdrName } = response.body;
-  t.is(pdrName, fakePdrs[0].pdrName);
+  t.is(pdrName, response.body.pdrName);
```

Comment:
Wondering: should we not do a deep comparison of the response against the expected API record structure for a PDR? Since we want to be sure that our response is getting properly translated from PG -> API format?

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/7ed58e6a1895c91c2d21825e119ebfa9ace452bc